.stylelintrc.json

```diff
@@ -0,0 +1,33 @@
+{
+    "extends": "stylelint-config-standard",
+    "rules": {
+        "indentation": 4,
+        "string-quotes": "double",
+        "no-duplicate-selectors": true,
+        "color-hex-length": "short",
+        "color-named": "always-where-possible",
+        "selector-combinator-space-after": "always",
+        "selector-attribute-quotes": "always",
+        "selector-attribute-operator-space-before": "always",
+        "selector-attribute-operator-space-after": "always",
+        "declaration-no-important": true,
+        "declaration-colon-space-before": "never",
+        "declaration-colon-space-after": "always",
+        "property-no-vendor-prefix": true,
+        "value-no-vendor-prefix": true,
+        "function-url-quotes": "always",
+        "font-family-name-quotes": "always-unless-keyword",
+        "comment-whitespace-inside": "always",
+        "comment-empty-line-before": "always",
+        "at-rule-no-vendor-prefix": true,
+        "rule-empty-line-before": "always",
+        "selector-pseudo-element-colon-notation": "double",
+        "selector-no-vendor-prefix": true,
+        "media-feature-range-operator-space-before": "always",
+        "media-feature-range-operator-space-after": "always",
+        "media-feature-parentheses-space-inside": "never",
+        "media-feature-name-no-vendor-prefix": true,
+        "media-feature-colon-space-before": "always",
+        "media-feature-colon-space-after": "always"
+    }
+}
```

Comment:
Where did we get this standard from?

Reply:
It's the default generated.  We'll adapt it to not be too intrusive over time.